health-checker/src/main/java/io/stargate/health/StorageHealthChecker.java

```diff
@@ -92,7 +93,8 @@ protected Result check() throws Exception {
                   .ttl(INSERT_TTL_SECONDS)
                   .timestamp(writeTimestamp.toEpochMilli())
                   .build()
-                  .bind())
+                  .bind(),
+              ConsistencyLevel.LOCAL_QUORUM)
```

Comment:
I thought we wanted to write at `LOCAL_ONE` and read at `LOCAL_QUORUM`

Reply:
why not read and write at `LOCAL_QUORUM`?

Reply:
Yes, I made a mistake in my message above :facepalm: `LOCAL_QUORUM` is the right CL for both reads and writes.